.pre-commit-config.yaml

```diff
@@ -1,37 +1,32 @@
 exclude: "^conda.recipe"
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v2.3.0
+    rev: v3.4.0
     hooks:
       - id: check-toml
       - id: check-yaml
       - id: end-of-file-fixer
       - id: trailing-whitespace
 
-  - repo: https://github.com/prettier/pre-commit
-    rev: main
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: v2.2.1
     hooks:
       - id: prettier
 
-  - repo: https://github.com/asottile/seed-isort-config
```

Comment:
Actually, why we removed that ?
The purpose of this was to correctly distinguish 3rd party libs etc Does it still work like that in this PR ?

Reply:
I removed it after reading [this](https://pycqa.github.io/isort/docs/configuration/pre-commit/) page in the documentation. Apparently, they have incorporated the facilities internally.
And the tests are still passing in pre-commit